src/d3a/d3a_core/sim_results/area_statistics.py

```diff
@@ -546,23 +552,24 @@ def gather_rates_one_market(cls, area, market, price_lists):
         price_lists[area][market.time_slot_str].extend(trade_rates)
 
     def update(self, area):
-        self._price_energy_day = self.gather_trade_rates(
-            area, self._price_energy_day,
-            use_last_past_market=not ConstSettings.GeneralSettings.KEEP_PAST_MARKETS
-        )
         latest_output_area_names = self.gather_trade_rates(
```

Comment:
Why is this called latest_output_area_names again? Isn't this a dict that holds trade rates?
If I am right, please rename area_name_output, too.
Thanks a lot!

Reply:
Yes, the naming is bad here. I will rename latest_output_area_names to current_price_lists, and the area_name_output to price_energy_csv_output. Meanwhile I will delete the latest_output member and replace it by reusing the redis_output member.